### PR TITLE
Add information for srcset and picture. Closes #367

### LINKED ISSUE #367: Responsive images srcset attribute
Hello!  Any plans to support information for the image `srcset` attribute?

http://caniuse.com/#feat=srcset
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img

Let me know if there's anything I can do to help.

### PR DESCRIPTION
**Pull request by louh**

### CODE REVIEW

**Comment by callumlocke** [comment]
:+1: can someone review this and merge it in?

**Comment by louh** [comment]
Wow, it has been some time. I'm still paying attention to this thread; let me know how I can help or if there's anyone I should poke.

**Comment by mischah** [comment]
Hej @louh,
so sorry that it took ages until someone took notice of your very welcome PR.
There was definitely a lack of maintenance :see_no_evil:

I hope your are willing to update the PR (eg. Browser support has changed since your commit). 
Otherwise I would take care of that :kissing_heart:  
… just let me know.

File: posts/picture.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+feature: <picture>
+status: use
+tags: fallback noie
+kind: html
+polyfillurls: [PictureFill](http://scottjehl.github.io/picturefill/)
+moreurl: http://www.html5rocks.com/en/tutorials/responsive/picture-element/
+
+The `<picture>` element allows a designer to explicitly specify the right image to use for different scenarios, similar to using media queries but for image tag sources. The element itself is merely a container; any number of `<source>` tags inside of `<picture>` is where all the magic happens. Finally, a required `<img>` element declares the default image to use if no other conditions are met. Browsers that do not support `<picture>` will gracefully fallback to this `<img>` tag, making this safe to use.
+
+Firefox supports `<picture>` now as of version 38, but Internet Explorer and Safari [lacks support](http://caniuse.com/#feat=picture). A [polyfill](http://scottjehl.github.io/picturefill/) is available, but it is not required.
```
Comment (mischah) [inline posts/picture.md]: Needs Update. Things changed for the better :v:

Comment (mischah) [inline posts/picture.md]: Needs Update. Things changed for the better :v:

File: posts/img-srcset.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+If you know what size the image will display at, the optional `sizes` attribute helps the browser to decide the most appropriate image before waiting to reflow the entire DOM first. For example, `sizes="100vw"` means a full-width image. (Percentage values will not work here, since the browser will have to answer the question "percent of what?", which requires a full DOM reflow.)
+
+Because `srcset` is an attribute on the standard `<img>` tag, browsers that do not understand it fallback to the image specified in `src`, making this safe to use immediately. Firefox supports `srcset` now as of version 38, but Safari only has [partial support](http://caniuse.com/#feat=srcset) for resolution switching only. A [polyfill](http://scottjehl.github.io/picturefill/) is available if you want to bring this functionality to older browsers and Internet Explorer, but it is not required.
```
Comment (mischah) [inline posts/img-srcset.md]: Browser support needs update. Things changed for the better :v:

**Comment by louh** [comment]
Hey @mischah, sure, I can update this today. Stay tuned.

**Comment by louh** [comment]
Updated! Feel free to make edits for content, writing style, etc.

**Comment by mischah** [comment]
Thanks for the PR, your patience and the updates :octocat:

Your commit did it to the master branch and the change should be deployed in a matter of minutes.

**Pull request closed without merging**